third_party/silabs/efr32_sdk.gni

```diff
@@ -200,6 +200,18 @@ template("efr32_sdk") {
           "RSI_SPI_INTERFACE",
           "RSI_WITH_OS",
         ]
+      } else if (use_SiWx917) {
```

Comment:
as mentionned before. Both statement defines the same thing. Why split it in two and repeate code.

Reply:
Modified Now.